python/lib/dependabot/python/pipenv_runner.rb

```diff
@@ -71,11 +71,12 @@ def dependency_name
 
       def pipenv_env_variables
```

Comment:
Unrelated but seems like this should be a constant

Reply:
Right, maybe it took a parameter at some point or something 🤷‍♂️.